### ISSUE TITLE
Small typo in Adagrad documentation

### ISSUE THREAD

**Issue by justinkterry** [pytorch/pytorch#14924]
In this page https://pytorch.org/docs/stable/optim.html

It currently reads:
lr (float, optional) – learning rate (default: 1e-2)

It should read:
lr (float, optional) – learning rate (default: **10**e-2)

**Comment by SsnL** [comment]
1e-2 is standard python notation of 0.01. 10e-2 is 0.1.
